saleor/graphql/product/tests/test_product.py

```diff
@@ -4061,6 +4061,44 @@ def test_query_product_image_by_invalid_id(
     assert content["data"]["product"]["imageById"] is None
 
 
+@pytest.mark.parametrize(
+    "sort_by, ascending",
+    (
+        ({"field": "ID", "direction": "ASC"}, True),
+        ({"field": "ID", "direction": "DESC"}, False),
+        (None, True),
+    ),
+)
+def test_query_product_media_sorting(
+    staff_api_client, product_with_image_list, channel_USD, sort_by, ascending
+):
+    query = """
+        query Product($id: ID!, $channel: String, $sort_by: MediaSortingInput){
+            product(id: $id, channel: $channel){
+                media(sortBy: $sort_by){
+                    id
+                }
+            }
+        }
+    """
+
+    variables = {
+        "id": graphene.Node.to_global_id("Product", product_with_image_list.pk),
+        "channel": channel_USD.slug,
+        "sort_by": sort_by,
+    }
+
+    response = staff_api_client.post_graphql(query, variables)
+    content = get_graphql_content(response)
+    media = content["data"]["product"]["media"]
+    media1 = graphene.Node.from_global_id(media[0]["id"])
```

Comment:
AFAIK, `from_global_id` returns a tuple (`Type`, `Id`). Is it expected here that you take into account both values?
Shouldn't be, `_, media1 = graphene.Node....`